test/test_one_view_four_view.py

```diff
@@ -0,0 +1,146 @@
+            dict_pixluts = ImageLoading.get_pixluts(read_data_dict)
+
+            patient_dict_container.set("rois", self.rois)
+            patient_dict_container.set("raw_contour", dict_raw_contour_data)
+            patient_dict_container.set("num_points", dict_numpoints)
+            patient_dict_container.set("pixluts", dict_pixluts)
+            dict_thickness = ImageLoading.get_thickness_dict(dataset_rtss, read_data_dict)
+
+        if 'rtdose' in file_names_dict:
+            dataset_rtdose = dcmread(file_names_dict['rtdose'])
+
+        # Open the main window
+        self.main_window = MainWindow()
+
+
+@pytest.fixture(scope="module")
+def test_object():
+    """Function to pass a shared TestStructureTab object to each test."""
+    test = TestOneViewAndFourViewsHandling()
+    return test
+
+
+def test_one_view_handling(qtbot, test_object, init_config):
+    test_object.main_window.show()
+    test_object.main_window.action_handler.action_one_view.trigger()
+    assert isinstance(test_object.main_window.dicom_single_view, DicomView) is True
+    assert test_object.main_window.dicom_view.currentWidget() == test_object.main_window.dicom_single_view
+
+
+def test_one_view_zoom(qtbot, test_object, init_config):
+    test_object.main_window.show()
+    test_object.main_window.action_handler.action_one_view.trigger()
+    initial_zoom = test_object.main_window.dicom_single_view.zoom
+    test_object.main_window.action_handler.action_zoom_in.trigger()
+    assert (test_object.main_window.dicom_single_view.zoom == 1.05 * initial_zoom)
+    test_object.main_window.action_handler.action_zoom_out.trigger()
+    assert (test_object.main_window.dicom_single_view.zoom == initial_zoom)
+
+
+def test_four_view_handling(qtbot, test_object, init_config):
+    test_object.main_window.show()
+    test_object.main_window.action_handler.action_four_views.trigger()
+    assert isinstance(test_object.main_window.dicom_view_axial, DicomView) is True
+    assert isinstance(test_object.main_window.dicom_view_sagittal, DicomView) is True
+    assert isinstance(test_object.main_window.dicom_view_coronal, DicomView) is True
+    assert isinstance(test_object.main_window.dicom_four_views_layout, QGridLayout) is True
+    assert test_object.main_window.dicom_view.currentWidget() == test_object.main_window.dicom_four_views
+    assert test_object.main_window.dicom_view_axial.pos().x(), test_object.main_window.dicom_view_axial.pos().y() == (
+        0, 0)
+    assert test_object.main_window.dicom_view_sagittal.pos().x(), test_object.main_window.dicom_view_sagittal.pos().y() == (
+        0, 1)
+    assert test_object.main_window.dicom_view_coronal.pos().x(), test_object.main_window.dicom_view_coronal.pos().y() == (
+        1, 0)
+
+
+def test_four_view_zoom(qtbot, test_object, init_config):
+    test_object.main_window.show()
+    test_object.main_window.action_handler.action_four_views.trigger()
+
+    initial_zoom = test_object.main_window.dicom_view_axial.zoom
```

Comment:
I don't think this is the proper way to test it. What if the zoom in dicom_view_axial is wrong?
It should be like this:
`initial_zoom = 1
assert (test_object.main_window.dicom_view_axial.zoom == initial_zoom)
...
` 
Or you can put the default zoom value in all DicomView classes in `src.constant` as `INITIAL_ZOOM`, then import that constant here and check if the default zoom in 4 view is equal to `INITIAL_ZOOM` here.
If you choose to do that, please rename `src.constant` to `src.constants` as well.